spec/components/previews/login_button_component_preview.rb

```diff
@@ -1,10 +1,13 @@
 class LoginButtonComponentPreview < ButtonComponentPreview
+  # @after_render :inject_style
+
   # @!group Preview
```

Comment:
In testing, I wasn't seeing the `<style>` tag. I think this `@after_render` needs to be right before the method definition.

```suggestion
  # @!group Preview
  # @after_render :inject_style
```